### DISCUSSION TITLE
When serving locally, cannot get /username

### DISCUSSION THREAD

**Discussion by Milo123459** [lowlighter/metrics#143] [Q&A] [answered]
http://localhost:3000/Milo123459?template=classic&config.timezone=Europe%2FLondon&config.animated=true  when going here I get a 404, so I can't test my new features locally :(

**Comment by Milo123459** [comment]
^^ Port is serving 3000

**Comment by lowlighter** [comment]
Did you configure `settings.json` ?
404 may also be thrown if your GitHub token is missing

**Comment by Milo123459** [comment]
Yes I did

**Comment by lowlighter** [comment]
Could you paste your `settings.json` (without your tokens of course) ?
Also if you can paste some logs from the console it would be nice 🙂

**Comment by Milo123459** [comment]
It's settings.exampl.json just with token as a blank new token with no scopes and it has my name as the only one who can do it

**Comment by lowlighter** [comment]
what does `npm start` output?
If you're in debug mode you should also be able to see what web instance loaded and its queries

**Comment by Milo123459** [comment]
Whenever I make a request it says flushed cache for my GitHub name

**Comment by lowlighter** [comment, marked as answer]
Ensure that: 
- you renamed `settings.example.json` to `settings.json` and it's located in metrics root directory
- you specified a valid GitHub token and set `debug` to true
- after running `npm start`, your `settings.json` should be printed in whole along with the following messages:
```
metrics/setup > setup
metrics/setup > load settings.json
metrics/setup > load settings.json > success
```
It it's printing this instead:
```
metrics/setup > load settings.json > (missing)
```
Then you're running in no settings mode (i.e. same env as for GitHub action) and your server cannot work, even if it starts.

See this [section](https://github.com/lowlighter/metrics#%EF%B8%8F-deploying-your-own-web-instance-15-min-setup-depending-on-your-sysadmin-knowledge) for more infos 👍
